### PR TITLE
fix sim_model_delay_steer_acc_geard model

### PR DESCRIPTION
**Pull request by hakuturu583**
Signed-off-by: MasayaKataoka <ms.kataoka@gmail.com>

## Types of PR

- [ ] New Features
- [ ] Upgrade of existing features
- [x] Bugfix

## Description

fix sim_model_delay_steer_acc_geard model

## How to review this PR.

Please check passing all CI test.

### CODE REVIEW

File: simulation/traffic_simulator/include/traffic_simulator/vehicle_model/sim_model_delay_steer_acc_geared.hpp [hunk 1]
```diff
@@ -110,7 +110,7 @@ class SimModelDelaySteerAccGeared : public SimModelInterface
   float64_t getVy() override;
 
   /**
-   * @brief get vehicle longitudinal acceleration
+   * @brief get vehicle longiudinal acceleration
```
Comment (yamacir-kit) [inline simulation/traffic_simulator/include/traffic_simulator/vehicle_model/sim_model_delay_steer_acc_geared.hpp:113]: Please correct this typographical error.

File: simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp [hunk 2]
```diff
@@ -120,17 +118,29 @@ float64_t SimModelDelaySteerAccGeared::calcVelocityWithGear(
...
+    state(IDX::VX) = 0.0;
+    state(IDX::X) = prev_state(IDX::X);
+    state(IDX::Y) = prev_state(IDX::Y);
+    state(IDX::YAW) = prev_state(IDX::YAW);
   }
 
-  return state(IDX::VX);
-}
+  state(IDX::ACCX) = (state(IDX::VX) - prev_state(IDX::VX)) / std::max(dt, 1.0e-5);
+}
```
Comment (yamacir-kit) [inline simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp:146]: Add newline to the end of the file using a suitable text editor.

**Review by yamacir-kit — CHANGES_REQUESTED** [review]

File: simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp (hunk 2 above)
Comment (hakuturu583) [reply simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp:146]: Fixed in https://github.com/tier4/scenario_simulator_v2/pull/707/commits/2ec83d51597ce0bfe63f09d5fb0deddb5789cc86

Comment (yamacir-kit) [reply simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp:146]: It hasn't been fixed.

**Review by yamacir-kit — CHANGES_REQUESTED** [review]

File: simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp (hunk 2 above)
Comment (hakuturu583) [reply simulation/traffic_simulator/src/vehicle_model/sim_model_delay_steer_acc_geared.cpp]: Sorry fixed in https://github.com/tier4/scenario_simulator_v2/pull/707/commits/dde2ff3a6c16a24f4693e2b98da8b1a0186840c7

**Review by yamacir-kit — APPROVED** [review]

**Pull request merged**